en/docs/observability/observability-overview.md

```diff
@@ -0,0 +1,72 @@
+# Observability Overview
+
+Observability in Choreo provides the capability to visualize and monitor the performance of services deployed on Choreo. Choreo has in-built support to view the overall status, latencies, throughput data,  diagnostic data, and logs. Hence, you can use Choreo Observability to detect and troubleshoot anomalies in your services efficiently. 
+
+## Dashboard
+Choreo observability dashboard gives you more than one way to monitor your services. This section will introduce the dashboards. 
+
+### Dashboard overview
+
+![Dashboard Overview](../assets/img/observability/overview-overall.png){: style="height:55%;width:95%"}
+
+The Observability dashboard allows you to:
+
+1. Observe the throughput and latencies of requests served over a given period.
+2. View the logs generated over a given period.
+3. Observe the flame graph (Diagnostics view) generated over a given period.
+4. View the low-code diagram.
+5.  Trace requests.
+
+!!! note
+    By default, the observability view is private and is only visible to the service owner. If you need to share it, toggle the **Share On** switch, copy the observability URL and share it with the required parties.
+    
+    ![Options Bar](../assets/img/observability/options-bar.png){: style="height:120%;width:90%"}
+
+### Throughput and latency graphs
+
+![Throughput and Latency Graph](../assets/img/observability/throughput-and-latency.png){: style="height:45%;width:70%"}
+
+The throughput graph shows the throughput of requests per hour for a selected timestamp. 
+
+By default, Choreo renders this graph for the data generated within the past 24 hours. You can change the default time window by selecting the time range and zone from the options bar. 
+
+You can expand the graph by clicking and dragging the cursor over the period you want to drill down.
+
+You can view the Choreo service logs in the **Logs**  view below the graph. Clicking on either graph will update the **Logs** view to contain the corresponding log entries generated at that time. You can use these logs to identify any evident reasons for any latency and throughput anomalies you detect using the graph. 
```

Comment:
`You can use these logs to identify any evident reasons for any latency and throughput anomalies you detect using the graph.` Can we avoid the repetition of `any` here?

Reply:
done